### PR TITLE
KEYCLOAK-6304: Czech translation

### PR DESCRIPTION
**Pull request by jakubknejzlik**
Add czech translations based on previous PR:
https://github.com/keycloak/keycloak/pull/4925

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+requiredAction.CONFIGURE_TOTP=Konfigurace OTP
+requiredAction.terms_and_conditions=Smluvní podmínky
+requiredAction.UPDATE_PASSWORD=Aktualizace hesla
+requiredAction.UPDATE_PROFILE=Aktualizace profilu
+requiredAction.VERIFY_EMAIL=Ověření e-mailu
+
+# units for link expiration timeout formatting
+linkExpirationFormatter.timePeriodUnit.seconds=sekundy
+linkExpirationFormatter.timePeriodUnit.seconds.1=sekunda
+linkExpirationFormatter.timePeriodUnit.minutes=minuty
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties:39]:
```suggestion
linkExpirationFormatter.timePeriodUnit.minutes=minut
```

File: themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+eventUpdateTotpBodyHtml=<p>V účtu {0} bylo změněno nastavení OTP z {1}. Pokud jste to nebyli vy, kontaktujte administrátora.</p>
+
+requiredAction.CONFIGURE_TOTP=Konfigurace OTP
+requiredAction.terms_and_conditions=Smluvní podmínky
+requiredAction.UPDATE_PASSWORD=Aktualizace hesla
+requiredAction.UPDATE_PROFILE=Aktualizace profilu
+requiredAction.VERIFY_EMAIL=Ověření e-mailu
+
+# units for link expiration timeout formatting
+linkExpirationFormatter.timePeriodUnit.seconds=sekundy
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties:37]:
```suggestion
linkExpirationFormatter.timePeriodUnit.seconds=sekund
```
and alternatives for 2, 3, and 4 (similarly to what you have for _minutes_ below)

File: themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+linkExpirationFormatter.timePeriodUnit.seconds.1=sekunda
+linkExpirationFormatter.timePeriodUnit.minutes=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.1=minuta
+linkExpirationFormatter.timePeriodUnit.minutes.2=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.3=minuty
+linkExpirationFormatter.timePeriodUnit.minutes.4=minuty
+linkExpirationFormatter.timePeriodUnit.hours=hodiny
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties:44]:
```suggestion
linkExpirationFormatter.timePeriodUnit.hours=hodin
```
and alternatives for 2, 3, and 4 (similarly to what you have for _minutes_ above)

File: themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+linkExpirationFormatter.timePeriodUnit.hours.1=hodina
+linkExpirationFormatter.timePeriodUnit.days=dny
```
Comment (hmlnarik) [inline themes/src/main/resources-community/theme/base/email/messages/messages_cs.properties:46]:
```suggestion
linkExpirationFormatter.timePeriodUnit.days=dní
```
and alternatives for 2, 3, and 4 (similarly to what you have for _minutes_ above)

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Thank you, looks pretty good to me! Few minor comments and is good to go from my perspective
@vramik @mposolda Would you like to double check?

**Comment by jakubknejzlik** [comment]
@hmlnarik Thanks for such quick reply. I've updated the email translations according to Your suggestions

File: themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+doSave=Uložit
+doCancel=Zrušit
+doLogOutAllSessions=Odhlásit všechny relace
+doRemove=Odstranit
+doAdd=Přidat
+doSignOut=Odhlásit se
+
+editAccountHtmlTitle=Upravit účet
+federatedIdentitiesHtmlTitle=Propojené identity
+accountLogHtmlTitle=Účet účtu
```
Comment (vramik) [inline themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties:11]:
```suggestion
accountLogHtmlTitle=Log účtu
```

**Review by vramik — COMMENTED** [review]
Thank you @jakubknejzlik for the PR. It seems good to me, I've found just one small thing, otherwise it's good to go in.

File: themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties (hunk 5 above)
Comment (jakubknejzlik) [reply themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties]: Oh, good catch 👍 ...fixed

**Review by vramik — APPROVED** [review]
@jakubknejzlik thanks

File: themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+doSave=Uložit
+doCancel=Zrušit
+doLogOutAllSessions=Odhlásit všechny relace
+doRemove=Odstranit
+doAdd=Přidat
+doSignOut=Odhlásit se
+
+editAccountHtmlTitle=Upravit účet
+federatedIdentitiesHtmlTitle=Propojené identity
+accountLogHtmlTitle=Lod účtu
```
Comment (vramik) [inline themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties:11]: I'm sorry, but there is a typo. 
```suggestion
accountLogHtmlTitle=Log účtu
```

**Review by vramik — CHANGES_REQUESTED** [review]

File: themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties (hunk 6 above)
Comment (jakubknejzlik) [reply themes/src/main/resources-community/theme/base/account/messages/messages_cs.properties]: Oh, sorry for that :-/ ... fixed.

**Review by vramik — APPROVED** [review]

**Review by hmlnarik — APPROVED** [review]
@jakubknejzlik Thank you!

**Pull request merged**